pkg/controller/clone-controller.go

```diff
@@ -213,6 +213,14 @@ func (r *CloneReconciler) reconcileSourcePod(sourcePod *corev1.Pod, targetPvc *c
 			return false, err
 		}
 
+		sourcePopulated, err := PopulationComplete(sourcePvc, r.client)
+		if err != nil {
+			return false, err
+		}
+		if !sourcePopulated {
```

Comment:
Would this be making it suck that clone sources would have to be DataVolumes?  If so, that's not cool.

Reply:
again I was confused

Reply:
No regular PVCs without a DV will already be marked as sourcePopulated = true